charts/dnsmadeeasy-webhook/README.md

```diff
@@ -0,0 +1,135 @@
+# dnsmadeeasy-webhook
+
+![Version: 2.0.0](https://img.shields.io/badge/Version-2.0.0-informational?style=flat-square) ![AppVersion: 1.0.0](https://img.shields.io/badge/AppVersion-1.0.0-informational?style=flat-square)
+
+Cert-Manager Webhook for DNSMadeEasy
+
+**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/k8s-at-home/charts/issues/new/choose)**
+
+## Source Code
+
+* <https://github.com/k8s-at-home/dnsmadeeasy-webhook>
+* <https://cert-manager.io>
```

Comment:
Maybe explicitly mention somewhere that a working cert-manager install is required for this to function?

Reply:
Added to charts/dnsmadeeasy-webhook/README.md.gotmpl -> Dependencies